terraform/fec_rds.tf

```diff
@@ -100,7 +99,7 @@ resource "aws_db_instance" "rds_production" {
 }
 
 resource "aws_db_instance" "rds_production_replica_1" {
-  replicate_source_db = "${aws_db_instance.rds_production.identifier}"
+  replicate_source_db = "fec-govcloud-prod"
```

Comment:
This is correct, but IMO it's clearer to stick with the reference instead of the literal name in case the identifier ever changes.